.github/workflows/markdownlint.yml

```diff
@@ -0,0 +1,22 @@
+name: MarkdownLint
+on: pull_request
+
+jobs:
+  lint:
```

Comment:
Copy paste this job and have 2 jobs: one for linting lesson files and another for project files